extension/content.js

```diff
@@ -60,6 +62,25 @@ function addToggleBtn() {
 	}
 }
 
+function fixupOtherButtons() {
+	const sidebar = document.querySelector('.file-navigation .right');
+	if (!sidebar) {
+		return;
+	}
+
+	const buttons = Array.from(sidebar.getElementsByClassName('btn btn-sm'));
+	buttons.forEach(button => {
```

Comment:
> I ran it the code in the profiler and fixupOtherButtons consistently ran in sub-millisecond time.

And querySelectorAll didn't? Anyways. Doesn't really matter :) Just would be more consistent and qSA should be as fast as `getElementsByClassName`.